### PR TITLE
Fix datetime format

### PR DESCRIPTION
**Pull request by matheusvill**

### CODE REVIEW

**Comment by julianogalgaro** [comment]
nice!

**Review by julianogalgaro — APPROVED** [review]
nice fix! Following the specification :)

**Comment by ppizarro** [comment]
LGTM

File: format_checkers_test.go [hunk 1]
```diff
@@ -26,6 +26,15 @@ func TestURIReferenceFormatCheckerIsFormat(t *testing.T) {
 	assert.True(t, checker.IsFormat("https://dummyhost.com/dummy-path?dummy-qp-name=dummy-qp-value"))
 }
 
+func TestDateTimeFormatCheckerIsFormat(t *testing.T) {
+	checker := DateTimeFormatChecker{}
+
+	assert.False(t, checker.IsFormat("2019-01-01"))
+	assert.False(t, checker.IsFormat("2019-01-01 10:00:00"))
+	assert.False(t, checker.IsFormat("2019-01-01T10:00:00"))
+	assert.True(t, checker.IsFormat("2019-01-01T10:00:00Z"))
```
Comment (ppizarro) [inline format_checkers_test.go:35]: We could add a millisecond test to test the RFC3339Nano. What do you think?

**Pull request merged**